Localization/Translations/Zones/Kalimdor.lua

```diff
@@ -225,20 +225,20 @@ local kalimdorLocales = {
         ["koKR"] = "썬더 블러프",
         ["esMX"] = "Cima del Trueno",
         ["enUS"] = true,
-        ["frFR"] = "Thunder Bluff",
+        ["frFR"] = "Les Pitons-du-Tonnerre",
```

Comment:
Hmmm... This is a hard one. In Vanilla this is "Thunder Bluff" and got a localized name only later (WotLK?).

Reply:
Scratch that. We are using the post vanilla translations all over the place already 😂 🤷🏻